src/share/types.hpp

```diff
@@ -369,6 +373,7 @@ class types final {
 
           // Extra
           {"fn", key_code::vk_fn_modifier},
+          {"hyper", key_code::vk_hyper_modifier}, // equal to ctrl + option + cmd
```

Comment:
@mafredri That's an interesting use case. Anyway, Karabiner has it mapped to ⇧Shift+⌃Control+⌥Option (alt)+⌘Command

https://github.com/tekezo/Karabiner/blob/master/src/core/server/Resources/include/checkbox/standards/fn.xml#L103